source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -893,7 +893,7 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
       /**
        *  The ID of the Connection
        */
-      connectionId: number;
```

Comment:
Valid points about it being driver-generated and thus relatively inconsequential if it rolls over. However Java doesn't support uint32 whereas it does support int64. Plus there is an argument to be made that `connectionId` and `serverConnectionId` use the same type - and the server represents `serverConnectionId` as an `int64` even if it sometimes serializes it as a narrower type. As for the potential for rollover resulting in negative `connectionIds`, let's say we create a billion new connections per second (totally unrealistic), it would take over 292 years to roll over an `int64`:
```
int64.maxValue/60 seconds per minute/60 minutes per hour /24 hours per day/365 days per year/1,000,000,000 connections per second = 292 years
```

Thus I would argue that we should use `int64` for both `connectionId` and `serverConnectionId` for consistency.